packages/oauth/src/providers/slack.ts

```diff
@@ -44,7 +44,7 @@ export class SlackAuth<_Auth extends Auth = Auth> extends OAuth2ProviderAuth<
 			"https://slack.com/openid/connect/authorize",
 			{
 				clientId: this.config.clientId,
-				scope: ["oidc", "profile", ...scopeConfig],
+				scope: ["openid", "profile", "email", ...scopeConfig],
```

Comment:
I don't think `"email"` is required? 

```suggestion
				scope: ["openid", "profile", ...scopeConfig],
```

Reply:
I’m not 100% sure - but I think email is also required:

“You request the OpenID scopes—[openid](https://api.slack.com/scopes/openid), [email](https://api.slack.com/scopes/email), and [profile](https://api.slack.com/scopes/profile).”

https://api.slack.com/authentication/sign-in-with-slack

Reply:
> Your app will always request openid—the base scope you always need to request in any Sign in with Slack flow—and may request email and profile as well.